src/backend/mscoffobj.c

```diff
@@ -708,7 +725,8 @@ void MsCoffObj::term(const char *objfilename)
         return;
 #endif
 
-    build_syment_table();
+    bool bigobj = scnhdr_cnt > 0xffff;
```

Comment:
Are you sure this number is the number of symbols that get written to the current obj file?